### PR TITLE
Remove msgpack5 for .NET 6 /3

### LINKED ISSUE #22008: Update MessagePack article for .NET 6
Update this section for https://github.com/aspnet/Announcements/issues/454. The breaking change is doc'd here: https://docs.microsoft.com/en-us/dotnet/core/compatibility/aspnet-core/6.0/messagepack-library-change.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: e807e138-f84c-e63e-ab37-eec0d554d3bb
* Version Independent ID: 3ac41883-3bc2-30ba-a47f-462196850c12
* Content: [Use MessagePack Hub Protocol in SignalR for ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/signalr/messagepackhubprotocol?view=aspnetcore-6.0#javascript-client)
* Content Source: [aspnetcore/signalr/messagepackhubprotocol.md](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/signalr/messagepackhubprotocol.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-signalr**
* GitHub Login: @bradygaster
* Microsoft Alias: **bradyg**

### PR DESCRIPTION
**Pull request by Rick-Anderson**
Fixes #22008
@fiyazbinhasan please review

[Internal review URL](https://review.docs.microsoft.com/en-us/aspnet/core/signalr/messagepackhubprotocol?view=aspnetcore-6.0&branch=pr-en-us-22738#javascript-client)

### CODE REVIEW

File: aspnetcore/signalr/messagepackhubprotocol.md [hunk 1]
```diff
@@ -81,17 +81,15 @@ After installing the npm package, the module can be used directly via a JavaScri
 
 *node_modules\\@microsoft\signalr-protocol-msgpack\dist\browser\signalr-protocol-msgpack.js* 
 
-In a browser, the `msgpack5` library must be referenced. Use a `<script>` tag to create a reference. The library can be found at *node_modules\msgpack5\dist\msgpack5.js*.
+In a browser, `signalr-protocol-msgpack` must be referenced. Use a `<script>` tag to create a reference. The library can be found at *node_modules\msgpack5\dist\msgpack5.js*.
```
Comment (Rick-Anderson) [inline aspnetcore/signalr/messagepackhubprotocol.md:84]: Where is the library found?

Comment (BrennanConroy) [reply aspnetcore/signalr/messagepackhubprotocol.md:84]: Remove this section

Comment (Rick-Anderson) [reply aspnetcore/signalr/messagepackhubprotocol.md]: Removed, ready to review.

File: aspnetcore/signalr/messagepackhubprotocol.md [hunk 2]
```diff
@@ -81,18 +81,6 @@ After installing the npm package, the module can be used directly via a JavaScri
...
 
-In a browser, the `msgpack5` library must be referenced. Use a `<script>` tag to create a reference. The library can be found at *node_modules\msgpack5\dist\msgpack5.js*.
-
-The following javaScript files must be referenced in the order shown below:
-
-
-```html
-<script src="~/lib/signalr/signalr.js"></script>
-<script src="~/lib/msgpack5/msgpack5.js"></script>
-<script src="~/lib/signalr/signalr-protocol-msgpack.js"></script>
```
Comment (BrennanConroy) [inline aspnetcore/signalr/messagepackhubprotocol.md:92]: Should probably still keep part of this to show how to reference the library.
Just signalr.js and signalr-protocol-msgpack.js are needed

**Review by fiyazbinhasan — APPROVED** [review]

File: aspnetcore/signalr/messagepackhubprotocol.md [hunk 3]
```diff
@@ -154,6 +149,7 @@ The MessagePack protocol doesn't provide a way to encode the `Kind` value of a `
 
 ### DateTime.MinValue is not supported by MessagePack in JavaScript
 
+<!-- -Review msgpack5 below -->
```
Comment (BrennanConroy) [inline aspnetcore/signalr/messagepackhubprotocol.md:152]: I'll try to look into this today and get back to you on what we should do

Comment (BrennanConroy) [reply aspnetcore/signalr/messagepackhubprotocol.md:152]: Ok, this section can be removed for 6.0

File: aspnetcore/signalr/messagepackhubprotocol.md [hunk 4]
```diff
@@ -316,7 +299,7 @@ HubConnection messagePackConnection = HubConnectionBuilder.create("YOUR HUB URL
     .build();
 ```
 
-## MessagePack quirks
+## MessagePack considerations
```
Comment (BrennanConroy) [inline aspnetcore/signalr/messagepackhubprotocol.md:302]: Any reason this isn't changed for the other tfms?

**Review by BrennanConroy — APPROVED** [review]

File: aspnetcore/signalr/messagepackhubprotocol.md (hunk 4 above)
Comment (Rick-Anderson) [reply aspnetcore/signalr/messagepackhubprotocol.md:302]: > Any reason this isn't changed for the other tfms?

 Traffic Flow Management System ?   Transportation Financial Management System ?

I'll fix this in another PR as soon as I figure out what TFMS means

Comment (Rick-Anderson) [reply aspnetcore/signalr/messagepackhubprotocol.md:302]: Target Framework Monikers
got it.

Comment (Rick-Anderson) [reply aspnetcore/signalr/messagepackhubprotocol.md:302]: Once 6.0 ships, no one reads the other versions. Unless it's a glaring problem, we typically don't do ***nice to have** fixes.

Comment (BrennanConroy) [reply aspnetcore/signalr/messagepackhubprotocol.md:302]: Ok, but didn't you only change it in the 5.0 one? `::: moniker range=">= aspnetcore-5.0 < aspnetcore-6.0"`

Comment (Rick-Anderson) [reply aspnetcore/signalr/messagepackhubprotocol.md:302]: Good catch. Another causality of ***Moniker Hell***

Fixed them all in #22757
Jargon/slang doesn't MT (machine translate).

**Pull request merged**